### ISSUE TITLE
[Update] skin.rapier 9.6.0

### ISSUE THREAD

**Issue by gade01** [xbmc/repo-skins#72]
### Add-on details:
- **General**
  - Add-on name: Rapier
  - Add-on ID: skin.rapier
  - Version number: 9.6.0
  - Kodi/repository version: Jarvis
- **Code location**
  - URL: https://github.com/gade01/Rapier
  - Revision/tag: latest
  - Branch: Jarvis

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-skins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

**Comment by BigNoid** [comment]
Thx for the update!
Could you check the following:
```
- Include not defined: CustomPanelItemlayoutLabel - 720p/script-nextup-notification-NextUpInfo.xml: line 285
- Include not defined: CustomPanelFocusedlayoutLabel - 720p/script-nextup-notification-NextUpInfo.xml: line 309
- Include not defined: CustomPanelItemlayoutLabel - 720p/script-nextup-notification-StillWatchingInfo.xml: line 258
- Include not defined: CustomPanelFocusedlayoutLabel - 720p/script-nextup-notification-StillWatchingInfo.xml: line 282
```

**Comment by gade01** [comment]
Great catch!

Copy/paste error. :)

Fixed with: fbdcb31914b4d36227ad54a186dd28b9ee9f0514

**Comment by BigNoid** [comment]
Thx, updated with 03551fd0bc5902d3bff9241e6f1184b0ce77fa8a
